### PR TITLE
LPS-118086 Hashing Framework API

### PR DESCRIPTION
**Pull request by csierra**
/cc @arthurchan35

hey Brian, 
resending here with updated names in the methods.

Bests. 

Carlos.

### CODE REVIEW

File: modules/apps/portal-crypto-hash/portal-crypto-hash-api/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+hash-pepper-storage-select=Hash Pepper Storage OSGi Target Filter Selector
```
Comment (liferay-appsec) [inline modules/apps/portal-crypto-hash/portal-crypto-hash-api/src/main/resources/content/Language.properties:1]: Check that added properties are either titles or sentences

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by csierra** [comment]
ci:forward

**Comment by arthurchan35** [comment]
ci:test:relevant

**Comment by arthurchan35** [comment]
ci:stop

**Comment by arthurchan35** [comment]
ci is not responding, slack ci channel has related topic. Trying again.

**Comment by arthurchan35** [comment]
PR with fixed method name is sent here:
https://github.com/brianchandotcom/liferay-portal/pull/96868

**Pull request closed without merging**